pymc/model.py

```diff
@@ -466,51 +466,50 @@ def __init__(self, mean=0, sigma=1, name=''):
                 # will get model's name prefix
 
                 # 3) you can create variables with Var method
-                self.Var('v1', Normal.dist(mu=mean, sigma=sd))
-                # this will create variable named like '{prefix::}v1'
+                self.register_rv(pm.Normal.dist(mu=mean, sigma=sigma), 'v1', initval=1)
```

Comment:
```suggestion
                self.register_rv(Normal.dist(mu=mean, sigma=sigma), 'v1', initval=1)
```

Why are we adding the `pm` prefix and showing `initval`?